db/migrate/20170208080437_add_last_check_notifications_at_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastCheckNotificationsAtToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :last_check_notifications_at, :datetime, default: DateTime.now
```

Comment:
this default: DateTime.now not works as you expect
ALL new users will have SAME datimetime in DB (when this column will be added)